WcaOnRails/app/webpacker/components/EditSchedule/index.js

```diff
@@ -0,0 +1,183 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import {
+  Accordion,
+  Button,
+  Container,
+  Message,
+} from 'semantic-ui-react';
+
+import _ from 'lodash';
+
+import { useSaveWcifAction } from '../../lib/utils/wcif';
+import { changesSaved } from './store/actions';
+import wcifScheduleReducer from './store/reducer';
+import Store, { useDispatch, useStore } from '../../lib/providers/StoreProvider';
+import ConfirmProvider from '../../lib/providers/ConfirmProvider';
+import EditVenues from './EditVenues';
+import EditActivities from './EditActivities';
+
+function EditScheduleNew({
```

Comment:
At what point will "new" be removed from this component name?

Reply:
Until somebody smart like yourself realizes that I accidentally left it in there :innocent: